dietpi/dietpi-software

```diff
@@ -6783,6 +6793,19 @@ exec sudo -u $ha_user dash -c '$ha_pyenv_activation; exec pip3 install -U homeas
 
 		fi
 
+		software_id=191 # snapcast-client
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			INSTALL_URL_ADDRESS='https://github.com/badaix/snapcast/releases/download/v0.25.0/snapclient_0.25.0-1_without-pulse_armhf.deb'
```

Comment:
Did you test if these armhf binaries are compatible with ARMv6 (Raspberry Pi 1 and Zero)?

We usually have an auto-detection for the latest version based on GitHub API: https://api.github.com/repos/badaix/snapcast/releases/latest
This should work:
```
# Package architecture: arm64 is currently not available: https://github.com/badaix/snapcast/issues/706
local arch='armhf'
(( $G_HW_ARCH == 10 )) && arch='amd64'

local fallback_url="https://github.com/badaix/snapcast/releases/download/v0.25.0/snapclient_0.25.0-1_without-pulse_$arch.deb"
Download_Install "$(curl -sSfL 'https://api.github.com/repos/badaix/snapcast/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/snapserver_[^\"\/]*_$arch.deb\"/{print \$4}")"
```

Reply:
I've tested them on a Zero W assume that is the same? In fact that is all i've tested them on.

Reply:
Okay, yes that is fine. The maintainer builds the packages on Raspberry Pi OS 32-bit, so then indeed it is armv6hf and compatible with all RPi models 👍.

Reply:
Moved to auto detect the latest version for both client and server.